scripts/helm/install.sh

```diff
@@ -165,7 +165,10 @@ else
        --set="etcd.livenessProbe.initialDelaySeconds=5,etcd.readinessProbe.initialDelaySeconds=5,etcd.replicaCount=1" \
        --set="obs.callhome.enabled=true,obs.callhome.sendReport=false,localpv-provisioner.analytics.enabled=false" \
        --set="eventing.enabled=true,nats.cluster.enabled=false,nats.cluster.replicas=1" \
+       --set="loki.loki.commonConfig.replication_factor=2,loki.singleBinary.replicas=2,loki.minio.replicas=2" \
```

Comment:
Can we set the simplest loki config here?

Reply:
Yes we should do that here, since this script is meant for faster testing and not e2e (etcd and nats are already set to 1)